conan_provider.cmake

```diff
@@ -588,9 +588,8 @@ macro(conan_provide_dependency method package_name)
             message(STATUS "CMake-Conan: Installing single configuration ${CMAKE_BUILD_TYPE}")
             conan_install(${_host_profile_flags} ${_build_profile_flags} ${CONAN_INSTALL_ARGS} ${generator})
         else()
-            message(STATUS "CMake-Conan: Installing both Debug and Release")
-            conan_install(${_host_profile_flags} ${_build_profile_flags} -s build_type=Release ${CONAN_INSTALL_ARGS} ${generator})
-            conan_install(${_host_profile_flags} ${_build_profile_flags} -s build_type=Debug ${CONAN_INSTALL_ARGS} ${generator})
+            message(STATUS "CMake-Conan: Installing single configuration ${CMAKE_BUILD_TYPE}")
```

Comment:
Did you check `CMAKE_BUILD_TYPE` has some value in case of Release multiconfiguration build? I suspect it is always empty. According to the [CMake documentation](https://cmake.org/cmake/help/latest/variable/CMAKE_CONFIGURATION_TYPES.html#variable:CMAKE_CONFIGURATION_TYPES) it is related to a single-config generators.

Reply:
You can see this line in manual runs

`-- CMake-Conan: Installing single configuration Debug` 

It is messaged using `message(STATUS "CMake-Conan: Installing single configuration ${CMAKE_BUILD_TYPE}")`